cmd/controller/app/options/options.go

```diff
@@ -92,8 +95,11 @@ type ControllerOptions struct {
 }
 
 const (
-	defaultAPIServerHost            = ""
-	defaultKubeconfig               = ""
+	defaultAPIServerHost              = ""
+	defaultKubeconfig                 = ""
+	defaultKubernetesAPIQPS   float32 = 10
+	defaultKubernetesAPIBurst         = 20
```

Comment:
Not sure if these defaults are correct: https://github.com/kubernetes/kubernetes/blob/63d4a032707d4e19719a58707540858f86d02f6f/staging/src/k8s.io/client-go/rest/config.go#L115-L121

Reply:
oops were still upped from a test i ran